tests/transforms/test_sign_expand.py

```diff
@@ -179,16 +182,17 @@ def cost(x):
             res = qml.execute(tapes, dev, qml.gradients.param_shift)
             return fn(res)
 
-        def cost_trad(x):
+        def cost_param_shift(x):
             tape.set_parameters(x, trainable_only=False)
             tapes, fn = qml.transforms.hamiltonian_expand(tape)
```

Comment:
Shouldn't you be using `qml.transforms.sign_expand`?

Reply:
This test in generas was a bit pointless as through the usage of qml.sparse_hamiltonian this transform is not diffable at the moment. So I removed the test, and added a "gradient=False" to the qml.transform. Might fix this at a point in the future!